### ISSUE TITLE
Is there a way to use it with the composition api?

### ISSUE THREAD

**Issue by AlexandruSerban142** [vuejs/vue-apollo#853]
@Akryum It really looks like the composition api + apollo is a nice combo but I didn't find a way to make it work together. Is there a way to use vue-apollo smart queries with the composition api? Or any other way? The only thing that i found is this [GithubRepo](https://github.com/Austio/vue-apollo-hooks), but it seems that it is using the old version of the composition api and also not maintaned like the Vue-Apollo is. Thanks

**Comment by Austio** [comment]
@Akryum still have that github repo on this one.  Did you want me to throw up a PR for how this would work with the new sample?  I'm very intersted in this b/c i think it would help my team and also the community at large, just don't want to duplicate efforts.

Thanks for all your apollo contributions :)

**Comment by Akryum** [comment]
Duplicate of #687

**Comment by JoeSchr** [comment]
Hi there, 

I'm currently trying to gobble together postgraphile+nuxt starter and want to use Typescript, vue-composition-api and vue-apollo & nuxt + a nuxt-postgraphile plugin, I created for it (with a lot of help of Benjie (from graphile ) and Atinux (from nuxt)).

Does anyone have by any chance a working example lying around of ts + vue-apollo + composition-api. 

I spent way too much time this weekend trying to get it to work, but I just don't know how e.g.  to make TS not complain about the apollo{} object in createComponent(). I feel once I passed this initial hurdle it would be a free highway from there on and really worth the hard time so far.
